google/cloud/bigquery/v2/minimal/internal/job_request_test.cc

```diff
@@ -36,12 +36,12 @@ using ::google::cloud::bigquery_v2_minimal_testing::MakePostQueryRequest;
 using ::google::cloud::testing_util::StatusIs;
 using ::testing::HasSubstr;
 
-auto static const kMinCreationTime = 1111111111111;
-auto static const kMaxCreationTime = 1111111111211;
-std::chrono::system_clock::time_point static const kMinTime{
-    std::chrono::milliseconds{kMinCreationTime}};
-std::chrono::system_clock::time_point static const kMaxTime{
-    std::chrono::milliseconds{kMaxCreationTime}};
+auto static const kMinCreationTimeMs = 1111111111111;
+auto static const kMaxCreationTimeMs = 1111111111211;
+auto static const kMinTime = std::chrono::system_clock::from_time_t(0) +
+                             std::chrono::milliseconds{kMinCreationTimeMs};
+auto static const kMaxTime = std::chrono::system_clock::from_time_t(0) +
+                             std::chrono::milliseconds{kMaxCreationTimeMs};
```

Comment:
You've now opened up space to call these `kMinCreationTime` and `kMaxCreationTime`.